Services/Interfaces/web_interface/models/backtesting.py

```diff
@@ -15,19 +15,32 @@
 #  License along with this library.
 import os
 import asyncio
+import ccxt
 
 import octobot_commons.logging as bot_logging
 import octobot.api as octobot_api
 import octobot_backtesting.api as backtesting_api
 import octobot_tentacles_manager.api as tentacles_manager_api
 import octobot_backtesting.constants as backtesting_constants
 import octobot_services.interfaces.util as interfaces_util
+import octobot_trading.constants as trading_constants
 import tentacles.Services.Interfaces.web_interface.constants as constants
 import tentacles.Services.Interfaces.web_interface as web_interface_root
 
+
 LOGGER = bot_logging.get_logger("DataCollectorWebInterfaceModel")
 
 
+def get_full_candle_history_exchange_list():
```

Comment:
I think we need to update the `__init__.py` as well to expose these 2 methods and be able to call them from `models.` in controllers

Reply:
It has been done but not in my git folder XD